src/asset-swapper/utils/market_operation_utils/bridge_source_utils.ts

```diff
@@ -146,58 +146,33 @@ export function getCurveInfosForPair(chainId: ChainId, takerToken: string, maker
                             [makerToken, takerToken].filter((v) => c.metaTokens?.includes(v)).length > 0),
                 ),
             );
-        case ChainId.Arbitrum:
-            return Object.values(CURVE_ARBITRUM_INFOS).filter((c) =>
-                [makerToken, takerToken].every(
-                    (t) =>
-                        (c.tokens.includes(t) && c.metaTokens === undefined) ||
-                        (c.tokens.includes(t) &&
-                            [makerToken, takerToken].filter((v) => c.metaTokens?.includes(v)).length > 0),
-                ),
-            );
         default:
             return [];
     }
 }
 
 export function getCurveV2InfosForPair(chainId: ChainId, takerToken: string, makerToken: string): CurveInfo[] {
+    const filterTokenInfos = function (CURVE_V2_CHAIN_INFOS: { [name: string]: CurveInfo }): CurveInfo[] {
```

Comment:
nit/optional: use camel case since it's not a constant?

Reply:
ack